components/hab/src/cli/hab/sup.rs

```diff
@@ -29,14 +33,15 @@ use habitat_core::{env::Config,
 use rants::{error::Error as RantsError,
             Address as NatsAddress};
 use std::{fmt,
-          io,
           net::{IpAddr,
                 SocketAddr},
           path::PathBuf,
           str::FromStr};
 use structopt::{clap::AppSettings,
                 StructOpt};
 
+const HAB_CTL_KEYS_CACHE: &str = "/hab/cache/keys/ctl";
```

Comment:
We might want to put this into `habitat_core::fs` for consistency.

Reply:
Addressed in https://github.com/habitat-sh/habitat/pull/7923/commits/4ed067a2222253a2740eeb5457fb51a34ba434cc